var/spack/repos/builtin/packages/hdf5-vol-log/package.py

```diff
@@ -21,12 +21,15 @@ class Hdf5VolLog(AutotoolsPackage):
     version("1.2.0", tag="logvol.1.2.0")
     version("1.1.0", tag="logvol.1.1.0")
 
-    depends_on("hdf5@1.13.2:")
+    depends_on("hdf5@1.13:1.13.2", when="@:1.3.0")
```

Comment:
Needs a new version as soon as possible that depends_on("hdf5@1.14:" ...)

Reply:
I was going to ask if this could just use 1.14 now. It looks like somewhere between the original work by @svenevs and this update that the log vol changed it's constraint so maybe it is agnostic to the breaking changes in 1.13.3?

Reply:
No, it's not agnostic.  There is a new version of https://github.com/DataLib-ECP/vol-log-based tagged 3 weeks ago that needs to be added to the hdf5-vol-log package.  The new version should be built with hdf5@1.14.0:, the three older versions with hdf5@1.13.2.  These are the changes to hdf5-vol-log/package.py:
+    version("1.4.0", tag="logvol.1.4.0")

-    depends_on("hdf5@1.13.2:")
+    depends_on("hdf5@1.14.0:", when="@1.4.0")
+    depends_on("hdf5@1.13.2", when="@:1.3.0")

I can create a PR for those changes unless you would rather add them to this one.

Reply:
I can add them here if they are ready. If I do would you mind if I removed all of the versions that used 1.13 and dropped the hdf5@1.13 versions as well? Or do we want to keep the hdf5@1.13:1.13 versions around for a little longer?

Reply:
If they aren't needed for e4s, dropping the hdf5@1.13 versions should be fine.

Reply:
I tested the changes - they should be ready.

Reply:
The 1.13 versions are still needed for h5bench, so I am going to leave them there for now.